gamemodes/terrortown/gamemode/client/cl_scoring.lua

```diff
@@ -412,7 +424,6 @@ function CLSCORE:Toggle()
 end
 
 bind.Register("toggle_clscore", function()
-	CLSCORE:ClearPanel()
```

Comment:
I couldnt find out why this was in here. But it had to be removed because of two things.
First, it was hiding the panel and therefore the toggle just showed the panel again.
Second, moving it to showpanel repositioned the panel everytime. I think its better if it always has the last position you gave it.